testlapack/dgeqp3.go

```diff
@@ -0,0 +1,118 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package testlapack
+
+import (
+	"math"
+	"math/rand"
+	"testing"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/floats"
+)
+
+type Dgeqp3er interface {
+	Dgeqp3(m, n int, a []float64, lda int, jpvt []int, tau, work []float64, lwork int)
+}
+
+func Dgeqp3Test(t *testing.T, impl Dgeqp3er) {
+	rnd := rand.New(rand.NewSource(1))
+	for c, test := range []struct {
+		m, n, lda int
+	}{
+		{1, 1, 0},
```

Comment:
Skipping Dlaqps for the moment, I think that these matrix sizes are too small to trigger the blocked code. Have you checked the coverage if you test only Dgeqp3, and Ilaenv for the minimum block size?